lesson_8/funstions.js

```diff
@@ -0,0 +1,119 @@
+/*Дано масив з елементами різних типів. Створити функцію яка вираховує середнє арифметичне лише числових елементів даного масиву.*/
+
+numbers = [1, 2, 3, 14, 'bla', 3.2]
+
+/**
+ * @param {array <any>} nums 
+ * 
+ * @returns array <number>
+ */
+
+
+function average(nums) {
+  result = [];
```

Comment:
Why do you still don't use let or const to create variables??

Reply:
because of PYTHON)))))